### PR TITLE
Update Cypress

### PR DESCRIPTION
**Pull request by orlandohohmeier**

### CODE REVIEW

**Comment by DanielMSchmidt** [comment]
I think you added the `package-json.lock` file by accident 🤔

**Comment by nLight** [comment]
cypress retest

**Comment by nLight** [comment]
Alfred test this

**Comment by nLight** [comment]
Alfred test this

**Comment by nLight** [comment]
Alfred test this

**Comment by nLight** [comment]
Alfred test this

File: src/js/utils/__tests__/DOMUtils-test.js [hunk 1]
```diff
@@ -74,46 +74,52 @@ describe("DOMUtils", function() {
     it("calculates left padding", function() {
       this.style.push("padding-left: 1px");
       global.document.body.innerHTML = buildElement(this.style);
+      const div = global.document.querySelector("div");
+      // jsdom doesn't calculate offsetWidth
+      Object.defineProperty(div, "offsetWidth", {
+        get: jest.fn(() => 101)
+      });
 
-      // jsdom doesn't calculate
```
Comment (Poltergeist) [inline src/js/utils/__tests__/DOMUtils-test.js]: image

File: tests/pages/services/ServiceFormModal-cy.js [hunk 2]
```diff
@@ -40,10 +40,10 @@ describe("Service Form Modal", function() {
         cy.get(".modal-full-screen").should("to.have.length", 1);
       });
 
-      it("Should Autofocus on the Service ID input field", function() {
+      it.skip("Should Autofocus on the Service ID input field", function() {
```
Comment (Poltergeist) [inline tests/pages/services/ServiceFormModal-cy.js]: Why are we skipping this and other Tests?

File: tests/pages/services/ServiceFormModal-cy.js [hunk 3]
```diff
@@ -1374,7 +1372,7 @@ describe("Service Form Modal", function() {
 
         it('Should add new set of form fields when "Add Environment Variable" link clicked', function() {
           // Key focused
-          cy.focused().should("have.attr", "name").and("eq", "env.0.key");
+          // cy.get('.form-control[name="env.0.key"]:focus');
```
Comment (Poltergeist) [inline tests/pages/services/ServiceFormModal-cy.js]: If not needed let's remove.

File: tests/pages/services/ServiceFormModal-cy.js [hunk 4]
```diff
@@ -1408,7 +1406,7 @@ describe("Service Form Modal", function() {
 
         it('Should add new set of form fields when "Add Label" link clicked', function() {
           // Key focused
-          cy.focused().should("have.attr", "name").and("eq", "labels.0.key");
+          // cy.get('.form-control[name="labels.0.key"]:focus');
```
Comment (Poltergeist) [inline tests/pages/services/ServiceFormModal-cy.js]: No need? Let's trash it!

File: tests/pages/services/ServiceFormModal-cy.js (hunk 2 above)
Comment (Poltergeist) [reply tests/pages/services/ServiceFormModal-cy.js]: Okay got it, https://github.com/cypress-io/cypress/issues/1176 Cypress currently has problems with the focus functionality could you please add a comment here and above all other skipped tests to document why they are skipped.

**Comment by DanielMSchmidt** [comment]
@Poltergeist Added the comments 👍

**Comment by nLight** [comment]
@Poltergeist 🙏

**Pull request merged**